lib/Optimizer/Optimization.h

```diff
@@ -0,0 +1,9 @@
+#include "glow/Graph/Graph.h"
```

Comment:
All header files need to have a guard against including it several times.

Reply:
The convention here is to use full path from lib, like OPTIMIZER_OPTIMIZATION_H.